bcmc/src/test/java/com/adyen/checkout/bcmc/BcmcComponentParamsMapperTest.kt

```diff
@@ -100,29 +75,86 @@ internal class BcmcComponentParamsMapperTest {
             )
         )
 
-        val params = BcmcComponentParamsMapper(overrideParams).mapToParams(bcmcConfiguration)
+        val params = BcmcComponentParamsMapper().mapToParams(bcmcConfiguration, overrideParams)
 
-        val expected = BcmcComponentParams(
+        val expected = getBcmcComponentParams(
             shopperLocale = Locale.GERMAN,
             environment = Environment.EUROPE,
             clientKey = TEST_CLIENT_KEY_2,
             isAnalyticsEnabled = false,
             isCreatedByDropIn = true,
-            isHolderNameRequired = false,
-            shopperReference = null,
-            isStorePaymentFieldVisible = false,
             amount = Amount(
                 currency = "USD",
                 value = 25_00L
             ),
-            isSubmitButtonVisible = true
         )
 
         assertEquals(expected, params)
     }
 
+    @ParameterizedTest
+    @MethodSource("sessionSetupConfigurationSource")
+    @Suppress("MaxLineLength")
```

Comment:
Hmm this is strange, we shouldn't get a `MaxLineLength` error when a method is between ` `` `

Reply:
It's strange, I added it after running `./gradlew check` and it gave me an error!